doc/user/installing-graalvm.md

```diff
@@ -67,6 +67,13 @@ and you should have about 10 GB of RAM available.
 $ gu rebuild-images polyglot libpolyglot
 ```
 
+To be able to do so, you may need to install the `native-image` module if you
```

Comment:
We use the term `component` for things that `gu` installs, could you change to that?

Reply:
I amended the commit to s/module/component, should be good to go.